R/ggthemr_reset.R

```diff
@@ -5,10 +5,12 @@
 ggthemr_reset <- function () {
   
   if (is_ggthemr_active()) {
-  
-    # removing scales
-    scales <- expand.grid(c('colour', 'fill'), c('continuous', 'discrete', 'gradient'))
-    rm(list = apply(scales, 1L, function (x) sprintf('scale_%s_%s', x[1L], x[2L])), envir = .GlobalEnv)
+    
+    # resetting scales
+    options("ggplot2.discrete.fill" = NULL)
```

Comment:
the code style of the package seems to use single quotes instead of double quotes -- let's stick with that?

Reply:
good point! Sorry I was careless